File: SpringBoot-SecKill-Scala/src/main/scala/cn/edu/jxnu/seckill/rabbitmq/SeckillMessage.scala
package cn.edu.jxnu.seckill.rabbitmq

import scala.language.implicitConversions
import scala.beans.BeanProperty
import cn.edu.jxnu.seckill.domain.SeckillUser

/**
 * RabbitMQ秒杀消息
 *
 * @author 梦境迷离.
 * @time 2018年5月21日
 * @version v1.0
 */
class SeckillMessage {

    @BeanProperty
    var user: SeckillUser = _

    @BeanProperty
    var goodsId: Long = _

}

File: seckill/src/main/java/cn/edu/jxnu/seckill/rabbitmq/SeckillMessage.java
package cn.edu.jxnu.seckill.rabbitmq;

import cn.edu.jxnu.seckill.domain.SeckillUser;
import lombok.Data;

@Data
public class SeckillMessage {
	private SeckillUser seckillUser;
	private long goodsId;

	@Override
	public String toString() {
		return "SeckillMessage{" + "seckillUser=" + seckillUser + ", goodsId=" + goodsId + '}';
	}
}
